build.gradle

```diff
@@ -13,7 +13,7 @@ repositories {
 
 sourceCompatibility = '1.8'
 group = 'org.opencadc'
-version = '1.1.46'
+version = '1.1.47'
```

Comment:
Please make this version `1.2.0` as it is a big enough change (non backwards compatible configuration).

Reply:
ok